Readme.md

```diff
@@ -248,6 +248,7 @@ so they can be created in a single update and can be re-created if any of them i
  - rebase on updated `master` to not undo other changes
  - figure out project name by converting the class name to snake-case
  - run `PROJECT=foo bundle exec rake kennel:update_datadog` to test changes for a single project (monitors: remove mentions while debugging to avoid alert spam)
+   - you can also use `PROJECT=foo:bar:...` if your work spans multiple projects
```

Comment:
prefer to split by `,` since `:` already has a use in kennel

Reply:
`,` feels more natural, whereas `:` with get the guarantee that the project id itself can't contain `:`.

Practically speaking if a project id doesn't match `/\A\w+\z/` then my answer would probably be "well don't do that then"

So sure, I'm fine with `,`